app/controllers/sellers_controller.rb

```diff
@@ -11,14 +11,17 @@ def index
       raise InvalidParameterError, query.errors.full_messages.to_sentence
     end
 
-    @sellers = Seller.order("#{query.sort_key} #{query.sort_order}")
-
-    sellers = @sellers.map do |seller|
+    Rails.cache.fetch("sellers", expires_in: 60.seconds) do
+      @sellers = Seller.order("#{query.sort_key} #{query.sort_order}")
+      if stale?(@seller)
+      sellers = @sellers.map do |seller|
```

Comment:
nit: spacing? you can run `rubocop -a path/to/this/file` to get the autoformatting